pkg/config/config.go

```diff
@@ -134,3 +138,15 @@ func GetPodLock(podName string) *sync.Mutex {
 	index := int(h.Sum32())
 	return &PodLocks[index%1024]
 }
+
+func MustGetWebPort() int {
+	value, exists := os.LookupEnv("JUICEFS_CSI_WEB_PORT")
+	if exists {
+		port, err := strconv.Atoi(value)
+		if err != nil {
+			klog.Fatalf("Fail to parse JUICEFS_CSI_WEB_PORT %s, %v", value, err)
+		}
+		return port
+	}
+	return 6060
```

Comment:
This is the default debug port for all component, it may conflict with each other, so we need try next possible one

Reply:
csi node & controller runs inside container so there's no risk of port conflict. the code is likely coped from somewhere else where things run on host with potential port conflicts.

since there's metrics to be scraped, better use a stable port because it's also referenced in prometheus scrape configs [here](https://github.com/juicedata/charts/pull/80/files#diff-5cfab4ababfd6053807af91951ba46f0f9d8d7d282207ed06cdbb3e7d5b2fbb9R16)